### DISCUSSION TITLE
有什么自定义的主题(比如博客文档)的具体说明吗

### DISCUSSION THREAD

**Discussion by lkzwc** [facebook/docusaurus#9121] [General]
求指教

**Comment by ahKevinXy** [comment]
按照官方文档 可以去操作，把主题替换就可以了

**Comment by kuizuo** [comment]
要自定主题的话 更多是在官方主题 [theme-classic](https://docusaurus.io/docs/api/themes/@docusaurus/theme-classic) 基础上通过[swizzing](https://docusaurus.io/docs/swizzling) 的方式来自定义主题。如果要重头实现一个的话，你可能就需要额外借助 `plugin-content-blog`、`plugin-content-docs` 插件来实现数据管理了。

**Reply by ahKevinXy** [reply]
我抄了你很多 @kuizuo

**Reply by kuizuo** [reply]
😂，不过我最近正在重构，设计一个专属的 leading 页。将 首页 (/) 与 博客页 (/blog)  分离开。

**Reply by ahKevinXy** [reply]
快更新啊, 最近打包时间 docusaurus 不知道为啥那么慢
